Bottom: c4d4417c97a11a02356f775d26c5e59e71c39e8a
Top:    5a00bf75136d28388605c06379e300b737af7818
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-11 15:32:19 +0800

Refresh of autofs-5.0.7-fix-potential-null-dereference-in-lookup_mount.patch

---

diff --git a/lib/cache.c b/lib/cache.c
index 2d1c2da..be4917b 100644
--- a/lib/cache.c
+++ b/lib/cache.c
@@ -685,6 +685,7 @@ void cache_update_negative(struct mapent_cache *mc,
 			   time_t timeout)
 {
 	time_t now = time(NULL);
+	struct mapent *me;
 	int rv = CHE_OK;
 
 	me = cache_lookup_distinct(mc, key);
